From 68c494b9ea56a8a2164545419c1574f489bf30d3 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Sat, 24 Feb 2018 11:12:22 +0800
Subject: [PATCH] arm64: rockchip_linux_config: enable camera sensors used by
 dts

As rk3399's sapphire board have used 5 camera sensors on
rk3399-sapphire-excavator-linux.dts, enable configure to support these.

Change-Id: I0afe26ad40aeeba889a556b4024a4798feeaf08e
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/configs/rockchip_linux_defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index a01953c4dcc7..704e0d31cd3c 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -293,6 +293,10 @@ CONFIG_V4L_MEM2MEM_DRIVERS=y
 CONFIG_VIDEO_ROCKCHIP_RGA=y
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
 # CONFIG_VIDEO_IR_I2C is not set
+CONFIG_VIDEO_OV4689=y
+CONFIG_VIDEO_OV7750=y
+CONFIG_VIDEO_ADV7181=y
+CONFIG_VIDEO_OV13850=y
 # CONFIG_DVB_AU8522_V4L is not set
 # CONFIG_DVB_TUNER_DIB0070 is not set
 # CONFIG_DVB_TUNER_DIB0090 is not set
-- 
2.35.3

